LevelEditor/SliderWidget.py

```diff
@@ -0,0 +1,197 @@
+from PySide.QtCore import *
+from PySide.QtGui import *
+from numpy import mean
+from collections import OrderedDict
+
+sliders = None
+
+
+class AdjustValues(QWidget):
+    valuesChanged = Signal(list)
+
+    def __init__(self, val_list, min_val, max_val):
+        super(AdjustValues, self).__init__()
+        self.val_list = val_list
+        self.min_val = min_val
+        self.max_val = max_val
+        self.slider_count = len(val_list)
+        self.slides = OrderedDict()
+
+        # Insert Sliders
+        self.wrap = QVBoxLayout()
+
+        for i in range(len(val_list)):
+            row = QHBoxLayout()
+            lab = QLabel(str(val_list[i]))
+            slide = QSlider(Qt.Horizontal)
+
+            # remove button
+            rem_button = QPushButton()
+            rem_button.setText("X")
+            rem_button.clicked.connect(self.remove_level(row))
+
+            # setting slider attributes
+            slide.setRange(min_val, max_val)
+            slide.setTickPosition(QSlider.TicksAbove)
+            slide.sliderMoved.connect(self.change_label(lab, slide))
+            slide.sliderMoved.connect(self.adjust_slides(slide))
+            slide.sliderMoved.connect(self.send_values)
+            slide.setValue(val_list[i])
+
+            # label = l, slider = s, position = p
+
+            d = {"l": lab, "s": slide, "p": val_list[i]}
+
+            self.slides[slide] = d
```

Comment:
Yes that's right. I needed a way to access all of the sliders, labels, and positions later on and creating multiple lists got messy. In the current code I never use the slide attribute of the dictionary because I can just use the key, but when I was calculating averages I though I would need it. I just left it there in case I needed it again.